docker/files/amplify/package.json.base

```diff
@@ -0,0 +1,12 @@
+{
+  "scripts": {
+    "amplify": "amplify"
+  },
+  "dependencies": {
+    "@aws-amplify/cli": AMPLIFY_VERSION,
+    "@graphql-codegen/cli": "^2.13.11",
+    "@graphql-codegen/typescript": "^2.8.1",
+    "@graphql-codegen/typescript-operations": "^2.5.6",
+    "@graphql-codegen/typescript-react-apollo": "^3.3.6"
+  }
```

Comment:
I think if I went down the `jq`/`sed` route we'd end up with a lot of boilerplate code (repeated for every dependency) so I took a slightly different approach:
- You define the dependencies in a simple textfile called `dependencies`
- Instead of running `jq`/`sed` and writing `package.json` directly, Docker image runs `generate-package-json` script that goes through every dependency and injects version from CDapp's `package.json`
- It merges the resolved dependencies with `package.base.json` which can still be used to define scripts and anything else

Let me know your thoughts...